archive/app/ui/NotFound.scala

```diff
@@ -1,8 +1,14 @@
 package ui
 
+import conf.Configuration
+import play.api.libs.json.{JsValue, Json}
 import rendering.Renderable
 
 object NotFound extends Renderable {
-  override def props = None
+  override def props: Option[JsValue] = Some(
+    Json.obj(
+      "beaconUrl" -> Configuration.debug.beaconUrl
```

Comment:
yeah this looks good to me for now. i guess that will almost design itself as we go along?